znoyder/cli.py

```diff
@@ -22,12 +22,84 @@
 from argparse import Namespace
 from argparse import RawDescriptionHelpFormatter
 from argparse import REMAINDER
+from argparse import _SubParsersAction
+from argparse import _UNRECOGNIZED_ARGS_ATTR
+from argparse import SUPPRESS
+from argparse import ArgumentError
 
 from znoyder import browser
 from znoyder import downloader
 from znoyder import finder
 from znoyder import generator
 from znoyder import templater
+from znoyder.lib import logger
+
+
+class NoDefaultOverrideSubparserAction(_SubParsersAction):
```

Comment:
Perhaps I would simply call it SubparserAction (or OverridenSuperarserAction?). But whatever you feel about it.

Reply:
Yes, that's probably a better name, I tried to be as explicit as possible, but it's too long, I'll change it

Reply:
Just to left a note for any visitors: a generally good practice is to put such details in commit message or docstring (which I can see you did, so I am satisfied)